data/pages-content/terms.md

```diff
@@ -130,39 +122,38 @@ difference between the discounted ticket and the standard ticket price.
 Conference tickets are not subject to the
 [EU regulations on distance selling and the 14-day cooling off period](https://europa.eu/youreurope/citizens/consumers/shopping/guarantees-returns/index_en.htm).
 Outside the above refund policy, we do not provide refunds on ticket purchases
-after the Refund End Date.
+after the Refund End Date, though we will always consider your particular circumstances.
```

Comment:
I would leave this as it was. I don't think we want to encourage people begging for refunds. Those who really need them will contact us anyway, as they have.